### PR TITLE
LPS-125493 Let screen reader to know about required fields and moving the focus to the first error field in the form so the user gets notified

### PR DESCRIPTION
**Pull request by antonio-ortega**
Related Issue: https://issues.liferay.com/browse/LPS-125493

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @antonio-ortega, can you please check if these failures are related to your changes? Thanks

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/Checkbox/__snapshots__/Checkbox.es.js.snap [hunk 1]
```diff
@@ -35,8 +35,7 @@ exports[`Field Checkbox has a helptext 1`] = `
       Type something
     </span>
     <span
-      aria-hidden="false"
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/Checkbox/__snapshots__/Checkbox.es.js.snap:38]: @antonio-ortega, can you please create a single commit to the snapshots regeneration?

Thanks.

**Review by natocesarrego — CHANGES_REQUESTED** [review]

**Comment by antonio-ortega** [comment]
Thanks for your correction @natocesarrego .

Resent liferay-forms#95

**Pull request closed without merging**